contrib/kafka-humio-gateway/main.go

```diff
@@ -0,0 +1,288 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"os/signal"
+	"sync"
+	"syscall"
+	"time"
+
+	"github.com/go-yaml/yaml"
```

Comment:
I assume this is working but going over to https://github.com/go-yaml/yaml it says the import part should be: 
gopkg.in/yaml.v2

I wonder if this could have some reprcussion in the future.

Reply:
It does, but it's also unclear who owns and maintains gopkg.in. I'd prefer to stick with GitHub as the anchor.